src/Extensions/Activities/IActivityListenerConfigurator.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.Activities
+{
+	/// <summary>
+	/// Configures behavior of ActivityListener to decide when activity should be created based on additional information about them
+	/// </summary>
+	public interface IActivityListenerConfigurator
+	{
+		/// <summary>
+		/// Decides if activity object events that were created using the activity source object should be listened or not
+		/// </summary>
+		bool ShouldListenTo(ActivitySource activity);
+
+		/// <summary>
+		/// Decides if creating System.Diagnostics.Activity objects with a specific data state is allowed
```

Comment:
>Decides if creating System.Diagnostics.Activity objects with a specific data state is allowed [](start = 6, length = 93)

I would suggest to elaborate more on the difference between the last two methods, they have the same comments.

Reply:
Copied from official doc :) https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.activitylistener?view=net-5.0
I have vague suspicions on difference, I'll try to investigate and improve comments.